app/models/concerns/i18n_enums.rb

```diff
@@ -28,8 +28,11 @@ def i18n_enum(attr, possible_values = nil,
 
       define_method(:"#{attr}_label") do |value = nil|
         value ||= send(attr)
-        translation = I18n.t("#{prefix}.#{value.to_s.downcase.presence || NIL_KEY}")
-        translation.ends_with?(NIL_KEY) ? nil : translation
+        begin
+          I18n.t("#{prefix}.#{value.to_s.downcase.presence || NIL_KEY}")
+        rescue I18n::MissingTranslationData
```

Comment:
Tritt dies auch in Production so auf? Ich gehe davon aus, dass wir nur in dev und test bei missing translations raisen.